test/unit/tech/tech.test.js

```diff
@@ -14,20 +21,10 @@ q.module('Media Tech', {
     this.clock = sinon.useFakeTimers();
     this.featuresProgessEvents = Tech.prototype['featuresProgessEvents'];
     Tech.prototype['featuresProgressEvents'] = false;
-    Tech.prototype['featuresNativeTextTracks'] = true;
```

Comment:
we may want to add tests for native text tracks (which these are mocking up) but they currently don't exist and I needed to test our implementation of text tracks.